helm/pay-wallet-values-uat.yaml

```diff
@@ -98,6 +98,9 @@ microservice-chart:
     OTEL_METRICS_EXPORTER: otlp
     OTEL_LOGS_EXPORTER: otlp
     OTEL_TRACES_SAMPLER: "always_on"
+    EXPIRATION_QUEUE_NAME: pagopa-d-itn-pay-wallet-cdc-queue
```

Comment:
Only a survivor :)

Reply:
solved in 19ee07a5c21a7a41756535144f44838e5e1cdc85